examples/combobox/js/grid-combo.js

```diff
@@ -94,12 +94,16 @@ aria.GridCombobox.prototype.handleInputKeyDown = function (evt) {
       );
     }
     else {
+      if (!this.shown) {
+        setTimeout((function () {
+          // On Firefox, input does not get cleared here unless wrapped in
```

Comment:
Do we know why this happens? Is it a bug in Firefox?

Reply:
I think this is an issue of using keydown, instead of the keyup event.   I was only asked to fix the ESC behavior, so I didn't think it would be good to make more changes than requested.

Reply:
Thank you @jongund, that's good call. let's address that as a separate issue if there is a better way to resolve it than what is currently implemented.